recipes/zbar/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+./configure --prefix=$PREFIX  --disable-video --without-gtk --without-python --without-qt --without-imagemagick --without-xv --without-x --without-xshm
```

Comment:
@CJ-Wright this will not build zbarcam
@kdmurray91 can zbarcam be packaged separately?

Reply:
I was just about to ask that.

Reply:
I'm not sure of how to package things separately with conda, if that's what you mean. Are there docs on this?

Additionally, trying to build zbarcam on my system (Debian sid) fails with an error suggesting to re-compile my kernel to support video4linux. That seem to be something that conda wouldn't be able to package as a dependency?

Reply:
I think that is a question for @conda-forge/staged-recipes